test/v2/integration/accelerateTx.js

```diff
@@ -0,0 +1,97 @@
+require('should');
+const co = require('bluebird').coroutine;
+
+const BitGoJS = require('../../../src');
+
+describe('accelerateTransaction', function() {
+  // latest.bitgo.com / otto-latest@bitgo.com
+  const accessToken = 'v2x0f5a14ab6aa4c3d4a420444ed6494157b7259de634ad3836ba748335812c6252';
+  // wallet "cpfptest2"
+  const walletId = '5c346da55cb4b705795c5176eb7c70cd';
+  const walletPassphrase = 'cpfptest2';
+  const coinName = 'tbtc';
+  const bitgo = new BitGoJS.BitGo({ env: 'latest', accessToken });
+
+  let wallet;
+
+  // wallet was funded with 10_000_000 sats,
+  // so this should be sufficient for 1000 txs with satAmount
+  const satAmount = 10000;
+
+  const baseFeeRate = 1;
+  const cpfpFeeRate = 10;
+
+  const getUnconfirmedTx = co(function *(wallet) {
+    const { address } = yield wallet.createAddress();
+    const { transfer } = yield wallet.send({
+      amount: satAmount,
+      address,
+      feeRate: baseFeeRate,
+      walletPassphrase
+    });
+    return transfer.txid;
+  }.bind(this));
+
+  const getConfirmedTx = co(function *(wallet) {
+    const { transfers } = yield wallet.transfers({ limit: 1, minConfirms: 1 });
+    return transfers[0].txid;
+  }.bind(this));
+
+  let unconfirmedTx;
+
+  before(co(function *() {
+    this.timeout(0);
+    console.log('unlocking account...');
+    yield bitgo.unlock({ otp: '0000000' });
+    console.log('getting wallet...');
+    wallet = yield bitgo.coin(coinName).wallets().get({ id: walletId });
+    (wallet === undefined).should.eql(false);
+
+    console.log('creating unconfirmed tx...');
+    unconfirmedTx = yield getUnconfirmedTx(wallet);
+    console.log('created tx', unconfirmedTx);
+    (unconfirmedTx === undefined).should.eql(false);
+  }));
+
+  it('can accelerate an unconfirmed tx', co(function *() {
+    console.log('acclerateTx...');
+    // FIXME: the tx is not found in the db when we go to access it
```

Comment:
I found waiting 1000ms on my system changes the error to `cannot build tx without available unspents`

I will add a delay with 2000ms to be on the safe side